src/Debug/Control.elm

```diff
@@ -201,60 +199,173 @@ stringTextarea initialValue =
         }
 
 
-{-| A `Control` that allows a Date (include date and time) input
-with a date picker UI.
+{-| A `Control` that allows a date and time input using the browser's date picker UI.
 -}
 date : Time.Zone -> Time.Posix -> Control Time.Posix
 date zone initialValue =
     let
+        initial =
+            Time.Extra.posixToParts zone initialValue
+
+        initialDateTime : String
         initialDateTime =
-            { year = Time.toYear zone initialValue
-            , month = Time.toMonth zone initialValue
-            , day = Time.toDay zone initialValue
-            , hour = Time.toHour zone initialValue
-            , minute = Time.toMinute zone initialValue
-            }
-
-        toPosix { year, month, day, hour, minute } =
-            Time.Extra.partsToPosix zone
-                { year = year
-                , month = month
-                , day = day
-                , hour = hour
-                , minute = minute
-                , second = 0
-                , millisecond = 0
-                }
+            String.fromInt initial.year
+                ++ "-"
+                ++ twoDigitMonth initial.month
+                ++ "-"
+                ++ twoDigit initial.day
+                ++ "T"
+                ++ twoDigit initial.hour
+                ++ ":"
+                ++ twoDigit initial.minute
     in
-    date_ (DateTimePicker.initialStateWithToday initialDateTime) initialDateTime
-        |> map toPosix
+    date_ initialDateTime
```

Comment:
:art: Now that `date_` is clearly meant to deal with only the `String` representation, I think maybe `dateField` or `dateInputField` would be a good name for it?